stdlib/ballerina-http/src/main/ballerina/http/caching_commons.bal

```diff
@@ -158,20 +163,20 @@ public type RequestCacheControl object {
 };
 
 documentation {
-    Cache control directives configuration for responses
-
-    F{{mustRevalidate}} Represents the must-revalidate directive
-    F{{noCache}} Represents the no-cache directive
-    F{{noStore}} Represents the no-store directive
-    F{{noTransform}} Represents the no-transform directive
-    F{{isPrivate}} Represents the private and public directives
-    F{{proxyRevalidate}} Represents the proxy-revalidate directive
-    F{{maxAge}} Represents the max-age directive
-    F{{sMaxAge}} Represents the s-maxage directive
-    F{{noCacheFields}} Optional fields for no-cache directive. If sending any of the listed fields in a response,
-                        they must validated with the origin server.
-    F{{privateFields}} Optional fields for private directive. A cache can omit the fields specified and store
-                        the rest ofthe response.
+    Configures cache control directives for a `Response`.
+
+    F{{mustRevalidate}} Sets the `must-revalidate` directive
+    F{{noCache}} Sets the `no-cache` directive
+    F{{noStore}} Sets the `no-store` directive
+    F{{noTransform}} Sets the `no-transform` directive
+    F{{isPrivate}} Sets the `private` and `public` directives
+    F{{proxyRevalidate}} Sets the `proxy-revalidate` directive
+    F{{maxAge}} Sets the `max-age` directive
+    F{{sMaxAge}} Sets the `s-maxage` directive
+    F{{noCacheFields}} Optional fields for the `no-cache` directive. Before sending a listed field in a response, it
+                       must be validated with the origin server.
+    F{{privateFields}} Optional fields for the `private` directive. A cache can omit the fields specified and store
+                       the rest of the response.
```

Comment:
remove periods at the end of fields?

Reply:
Periods are required here since it's a multiple sentence description

Reply:
yeah, got it!